docs/pages/modules/module-api.mdx

```diff
@@ -494,6 +495,105 @@ The implementation for three Either types is currently provided out of the box,
 
 </APIBox>
 
+## Native Classes
+
+<APIBox header="Module">
+
+A base class for a native module.
+
+#### Properties
+
+<APIMethod
+  name="appContext"
+  comment="Provides access to the [`AppContext`](#appcontext)."
+  returnTypeName="AppContext"
+  isProperty={true}
+  isReturnTypeReference={true}
+/>
+
+#### Methods
+
+<APIMethod
+  name="sendEvent"
+  comment="Sends an event with a given name and a payload to JavaScript. See [`Sending events`](#sending-events)"
+  returnTypeName="void"
+  parameters={[
+    {
+      name: 'eventName',
+      comment: 'The name of the JavaScript event',
+      typeName: 'string',
+    },
+    {
+      name: 'payload',
+      comment: 'The event payload',
+      typeName: 'iOS: [String: Any?]; Android: Map<String, Any?> | Bundle',
```

Comment:
```suggestion
      typeName: 'iOS: [String: Any?]; Android: Map<String, Any?> | Bundle',
```

This part I did not like, ideally we should use the platforms tags here and list return types in separate lines, but it might be a bit tricky when reusing APISection components. If this is hard to active we can just add `/n` or `<br />` for now.

Also we should list Android before iOS.